kvbc/include/Replica.h

```diff
@@ -243,7 +243,7 @@ class Replica : public IReplica,
   const bftEngine::ReplicaConfig &replicaConfig_;
   bftEngine::IReplica::IReplicaPtr m_replicaPtr = nullptr;
   std::shared_ptr<ICommandsHandler> m_cmdHandler = nullptr;
-  bftEngine::IStateTransfer *m_stateTransfer = nullptr;
+  std::unique_ptr<bftEngine::IStateTransfer> m_stateTransfer;
```

Comment:
- The header of the PR looks wrong - you are replacing State transfer pointer with a raw pointer.
- Why are you merging into master? we still have a dev branch.

Reply:
> No, other way round "change **to** unique_ptr"

@titusen 
Suggested header for the PR:
"Convert m_stateTransfer to a unqiue_ptr"